.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text eol=lf
+
+*.icns binary
+*.zip binary
+*.wav binary
+*.swp binary
```

Comment:
Do we actually have files of these types in our repo?

Reply:
My first search included some build results, therefore `*.swp` is superfluous, but the remaining types occur (even if only one time) in our repo.
* `./de.fu_berlin.inf.dpp.ui.widgetGallery/icons/alt_launcher.icns`
* `./de.fu_berlin.inf.dpp.ui/libs/swt-browser-improved-src.zip`
* `./de.fu_berlin.inf.dpp/assets/sounds/*.wav`